Integrations/integration-GoogleVault.yml

```diff
@@ -1408,26 +1744,160 @@ script:
+    description: 'Get the results of a given drive export '
+  - name: gvault-get-mail-results
+    arguments:
+    - name: bucketName
+      required: true
+      description: name of the bucket that holds the export
+    - name: viewID
+      required: true
+      description: the export view ID
+    - name: maxResult
+      default: true
+      description: Set the threshold of result to be returned (High threshold can
+        slow down your browser)
+      defaultValue: "30"
+    outputs:
+    - contextPath: GoogleVault.Matter.Export.Results.CC
+      description: CC of the mail
+      type: string
+    - contextPath: GoogleVault.Matter.Export.Results.BCC
+      description: 'BCC of the mail '
+      type: string
+    - contextPath: GoogleVault.Matter.Export.Results.DateSent
+      description: The date the email was sent
+      type: string
+    - contextPath: GoogleVault.Matter.Export.Results.From
+      description: The sender of the email
+      type: string
+    - contextPath: GoogleVault.Matter.Export.Results.Subject
+      description: The subject of the email
+      type: string
+    - contextPath: GoogleVault.Matter.Export.Results.DateReceived
+      description: The date the email was received
+      type: string
+    - contextPath: GoogleVault.Matter.Export.Results.To
+      description: The address the email was sent to
+      type: string
+    description: Get the results of a given mail export
+  - name: gvault-get-groups-results
+    arguments:
+    - name: bucketName
+      required: true
+      description: name of the bucket that holds the export
+    - name: viewID
+      required: true
+      description: the export view ID
+    - name: maxResult
+      default: true
+      description: Set the threshold of result to be returned (High threshold can
+        slow down your browser)
+      defaultValue: "30"
+    outputs:
+    - contextPath: GoogleVault.Matter.Export.Results.CC
+      description: CC of the message
+      type: string
+    - contextPath: GoogleVault.Matter.Export.Results.BCC
+      description: BCC of the message
+      type: string
+    - contextPath: GoogleVault.Matter.Export.Results.DateSent
+      description: The date the message
+      type: string
+    - contextPath: GoogleVault.Matter.Export.Results.From
+      description: The sender of the message
+      type: string
+    - contextPath: GoogleVault.Matter.Export.Results.Subject
+      description: The subject of the message
+      type: string
+    - contextPath: GoogleVault.Matter.Export.Results.DateReceived
+      description: The date the message was received
+      type: string
+    - contextPath: GoogleVault.Matter.Export.Results.To
+      description: The address the message was sent to
+      type: string
+    description: Get the results of a given group export
   dockerimage: demisto/gvault
   runonce: false
+releaseNotes: 'New commands: gvault-get-drive-results, gvault-get-mail-results, gvault-get-groups-results, download-export.
```

Comment:
'download-export' dose not exist in the code, probably meant `gvault-download-results`
re check the comment and the commands in the YML.
make sure we have all the new commands names in the release notes and don't have leftovers like `download-export`' that do not exist in the code but do exist in the YML docs